apps/global/src/hooks/useTranslation.tsx

```diff
@@ -115,6 +115,11 @@ const query = graphql`
               meetTheCollectiveTitle
               meetTheCollectiveDescription
             }
+            board {
+              pageTitle
+              meetTheCollectiveTitle
+              meetTheCollectiveDescription
```

Comment:
We will need to remove this here too since we are now removing this from the board page